sources/packages/backend/apps/api/src/route-controllers/application/application.aest.controller.ts

```diff
@@ -117,4 +124,65 @@ export class ApplicationAESTController extends BaseController {
       throw error;
     }
   }
+
+  /**
+   * Triggers manual reassessment for an application.
+   * Application cannot be archived and current assessment must be in completed status.
+   * @param applicationId application id.
+   */
+  @Roles(Role.AESTAdmin)
+  @Post(":applicationId/manual-reassessment")
+  @ApiNotFoundResponse({ description: "Application id not found." })
+  @ApiUnprocessableEntityResponse({
+    description:
+      `Application current assessment expected to be '${StudentAssessmentStatus.Completed}' to allow manual reassessment or ` +
+      "application cannot have manual reassessment after being archived.",
+  })
+  async manualReassessment(
+    @Body() payload: ManualReassessmentAPIInDTO,
+    @Param("applicationId", ParseIntPipe) applicationId: number,
+    @UserToken() userToken: IUserToken,
+  ): Promise<void> {
```

Comment:
`POST` usually returns the id of the created resource using `Promise<PrimaryIdentifierAPIOutDTO>`.